docker-compose.dev.yml

```diff
@@ -0,0 +1,41 @@
+version: "3.4"
```

Comment:
We should probably keep the dev and prod compose-files as aligned as possible. That said, that's less important if the prod file is going away soon anyway.

Reply:
I wanted to update the prod-version, but read on slack that the server that runs it doesn't accept newer docker-compose versions. So didn't want to change the prod-version.